setup.py

```diff
@@ -10,7 +10,7 @@
 from sys import version_info, exit
 # Define our list of installation dependencies
 DEPENDS = ["pyjwt<2.0.0", 
-           "snowflake-connector-python", 
+           "snowflake-connector-python<=2.2.7", 
```

Comment:
should be other way around, right? We want min version to be 2.2.7
>=

snowflake-connector-python>=2.2.7